README.md

```diff
@@ -29,6 +25,21 @@ From the root folder:
 yarn test
 ```
 
+#### Local Packaging
+
+Local packaging lets you publish a package locally and try it out. One common use case is making changes to the launchpad, packaging them, and trying the out in the debugger.
+
+```bash
+cd devtools-core/packages/devtools-modules
+vim package.json # bump the version and add `-rc1` e.g `0.0.6-rc1`
+npm pack # note, every time you make a change you'll bump rc
+cd ../../../debugger.html
+yarn add file:../devtools-core/packages/devtools-modules/devtools-modules-v0.0.6-rc1.tgz
```

Comment:
we could add a comment saying to copy the name from the result of `npm pack` also